frontend/src/Settings/Notifications/Notifications/Notification.js

```diff
@@ -139,9 +141,16 @@ class Notification extends Component {
             </Label>
         }
 
+        {
+          supportsOnApplicationUpdate && onApplicationUpdate &&
+            <Label kind={kinds.SUCCESS} >
+              On Application Update
```

Comment:
Translate (and knock out the rest while we're here)

ref https://github.com/Radarr/Radarr/blob/fda1ad237bc9d93e420f637ece0c09d524b8b44b/frontend/src/Settings/Notifications/Notifications/Notification.js#L131

Reply:
I went ahead and did them all.